### PR TITLE
Doc high connection workloads

### PR DESCRIPTION
**Pull request by bmckercher123**
This is a new How To that was reviewed and approved by Alex.

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 87c41ee6f4a84fec65d56a483cd793b61a17a7d6

https://deploy-preview-2604--calico.netlify.com

**Comment by bmckercher123** [comment]
Links verified.

**Comment by bmckercher123** [comment]
Additional edits from Alex.

File: master/security/high-connection-workloads.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+title: Enable extreme high-connection workloads 
+---
+
+### Big Picture
+
+Use a Calico network policy rule to bypass Linux conntrack for traffic to extreme high-connection workloads.
```
Comment (caseydavenport) [inline master/security/high-connection-workloads.md]: Might just be me, but I find "extreme high connection workloads" to be a somewhat awkward or confusing term.

How do you feel about something like "for traffic to workloads which serve an extremely high rate of connections" or "for workloads which serve an extremely large number of connections"?

Maybe just "extremely high-connection workloads"?

File: master/security/high-connection-workloads.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+### Value
+
+When extreme high-connection workloads exceed the number of connections that Linux conntrack can track, connections can be rejected or dropped. Calico network policy can be used to selectively bypass Linux conntrack for traffic to/from these types of workloads.
```
Comment (caseydavenport) [inline master/security/high-connection-workloads.md]: I think this should say "When the number of connections on a node exceeds..."

File: master/security/high-connection-workloads.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      source:
+        ports:
+          - 12211
+      destination:
+        selector: memcached == 'client'
+```
+### Above and beyond
+
+- [Calico GlobalNetworkPolicy](https://docs.projectcalico.org/v3.6/reference/calicoctl/resources/globalnetworkpolicy)
+- [Blog: When Linux conntrack is no longer your friend](https://www.tigera.io/blog/when-linux-conntrack-is-no-longer-your-friend/)
```
Comment (caseydavenport) [inline master/security/high-connection-workloads.md]: Does it make sense for this blog to be both "Before you begin" as well as "Above and beyond"?

File: master/security/high-connection-workloads.md (hunk 1 above)
Comment (bmckercher123) [reply master/security/high-connection-workloads.md]: I agree with this. Alex overruled my other suggestions. Let me try to persuade him to take a look again. Usually, we don't use hyper language like "extremely high" when high already indicates something above and beyond. So I need convincing that in conntrack tables people make some distinction in their minds between "high" and "extremely high".

Comment (bmckercher123) [reply master/security/high-connection-workloads.md]: How about this:
Use a Calico network policy rule to bypass Linux conntrack for traffic to workloads that experience extremely large number of connections.

File: master/security/high-connection-workloads.md (hunk 2 above)
Comment (bmckercher123) [reply master/security/high-connection-workloads.md]: Fixed

File: master/security/high-connection-workloads.md (hunk 3 above)
Comment (bmckercher123) [reply master/security/high-connection-workloads.md]: It doesn't hurt to have both places, but I removed the one at the end.

File: master/security/high-connection-workloads.md (hunk 1 above)
Comment (bmckercher123) [reply master/security/high-connection-workloads.md]: Fixed

**Comment by bmckercher123** [comment]
When I click Resolve conflicts button, I get spinning icon. I cannot see the conflict.

**Pull request merged**